src/lru.rs

```diff
@@ -68,13 +68,14 @@ impl<K: HasWeight + Ord + Hash + Clone, V: HasWeight> LruCache<K, V> {
                 // cleaup process (thereby keeping our reads fast and paying the
                 // cost on writes instead)
                 Option::None
-            },
+            }
 
             Some(entry) => {
                 // we found it and it hasn't expired. Since it's being used now
                 // we need to update its position in the LRU
                 let k2 = Arc::new(key.clone());
-                if ((*entry).used) != now { // only update it if the value would change
+                if ((*entry).used) != now {
+                    // only update it if the value would change
```

Comment:
this is also slightly strange, I would probably move the comment above, not below.